web_timeline/README.rst

```diff
@@ -0,0 +1,75 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+    :alt: License: AGPL-3
+
+===============
+Timeline Widget
+===============
+
+Define a new widget displaying events in an interactive visualization chart.
+
+The widget is based on the external library 
+http://visjs.org/timeline_examples.html
```

Comment:
You might want to add a section "Installation".
Not familiar with js but make sure to check the library before use or at module installation (ignore if not relevant)